multisense_ros/src/reconfigure.cpp

```diff
@@ -93,6 +94,12 @@ Reconfigure::Reconfigure(Channel* driver,
         roi_supported_ = true;
     }
 
+    if (system::DeviceInfo::HARDWARE_REV_MULTISENSE_C6S2_S27 == deviceInfo.hardwareRevision ||
+        system::DeviceInfo::HARDWARE_REV_MULTISENSE_S30 == deviceInfo.hardwareRevision)
```

Comment:
The new monocam will need this as well.

Reply:
Can we add that once we have hardware in hand?